test/test_emulation/__init__.py

```diff
@@ -92,12 +92,19 @@ def udev_rules(self):
     def server(self):
         p = None
         try:
-            p = subprocess.Popen('python -m ratbag_emu 2&>1 >/dev/null', shell=True)
+            stdout = open('ratbag-emu-log-stdout.txt', 'w')
+            stderr = open('ratbag-emu-log-stderr.txt', 'w')
```

Comment:
lesson learned a few times: always use something date/timestamped like `filename-2019-09-18-13:00:12`. Worst case you'd have to delete a bunch of files, best case you didn't just overwrite the last log that had important information in it.

Also, after opening this write the current date/time into the file too. That way it's preserved even where the filename changes (e.g. when you post on pastebin)